kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/NetworkAdminConfiguration.java

```diff
@@ -16,6 +16,7 @@
     CONFIG_MTU,
     CONFIG_AUTOCONNECT,
     CONFIG_DRIVER,
+    CONFIG_IPV4_L2ONLY_ENABLED,
```

Comment:
Doesn't this change imply an update of the exported package in manifest file?

Reply:
Made changes to the MANIFEST.MF